docs/tutorials/morpheusvm/intro.md

```diff
@@ -125,12 +121,26 @@ import (
 var _ chain.Action = (*Transfer)(nil)
 
 type Transfer struct {
-   To codec.Address `serialize:"true" json:"to"`
-   Value uint64 `serialize:"true" json:"value"`
-   Memo []byte `serialize:"true" json:"memo"`
+	// To is the recipient of the [Value].
+	To codec.Address `serialize:"true" json:"to"`
+
+	// Amount are transferred to [To].
+	Value uint64 `serialize:"true" json:"value"`
+
+	// Optional message to accompany transaction.
```

Comment:
We should make sure that any code we are asking users to copy-paste is formatted correctly